gcc/testsuite/gdc.test/runnable/sdtor.d

```diff
@@ -1,3 +1,4 @@
+// PERMUTE_ARGS: -unittest -O -release -inline -fPIC -g
```

Comment:
Looks like changes so far haven't caused the current failure, but the addition of `fPIC` here has unearthed an existing bug in NRVO.